analyzers/tests/SonarAnalyzer.UnitTest/Helpers/DiagnosticVerifierTest.cs

```diff
@@ -18,11 +18,10 @@
 * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 */
 
-extern alias csharp;
 using System;
-using csharp::SonarAnalyzer.Rules.CSharp;
 using FluentAssertions;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
+using CS = SonarAnalyzer.Rules.CSharp;
```

Comment:
The alias is not needed. You can have directly `using SonarAnalyzer.Rules.CSharp;` and remove the prefix from constructor invocations bellow. Please check the other files as well.

Reply:
I fixed all non Rule class cases. For the rule class tests the rspec generator should be changed too, so I put it on how we work to align what we want.